src/ui/instructions.ts

```diff
@@ -0,0 +1,92 @@
+import { Entity, TextureFilterMode, TransformTypeWithOptionals } from '@dcl/sdk/ecs'
+import { Quaternion, Vector3 } from '@dcl/sdk/math'
+
+import { MenuButton } from './button'
+import { getSDK } from '../sdk'
+
+export class InstructionsBoard {
+  textures: string[]
+  boardRoot: Entity
+  board: Entity
+  buttonLeft?: MenuButton
+  buttonRight?: MenuButton
+  height: number = 3.65
+  width: number = 6.3
+
+  constructor(transform: TransformTypeWithOptionals, width: number, height: number) {
+    const { engine, Transform, MeshRenderer, Material } = getSDK()
+    this.width = width
+    this.height = height
+    this.boardRoot = engine.addEntity()
+    Transform.create(this.boardRoot, transform)
+
+    this.textures = []
+    // TODO: this instructions should be passed as an argument.
```

Comment:
Yeah, agreed! I thought maybe the same filename will be enough, but it's better to pass this as an argument.

Reply:
I'm ok with the same filename for now